Image creation fails with string error in base_path

Reason could be the following updated code lines by [#521 ](https://github.com/opentelekomcloud/python-otcextensions/pull/521)
Version 0.32.8 is working but 0.32.9 is making trouble.

base_path
https://github.com/opentelekomcloud/python-otcextensions/blob/bee342028e159cdda24006798353524f5bae5dc1/otcextensions/sdk/imsv2/v2/image.py#L20

https://github.com/opentelekomcloud/python-otcextensions/blob/bee342028e159cdda24006798353524f5bae5dc1/otcextensions/sdk/imsv2/v2/image.py#L91

Error Message:
```
self = otcextensions.sdk.imsv2.v2.image.Image(name=06209a8c-8753-41a3-9d4b-d825406c7ea3, image_url=***MY_IMAGE_URL***', 'name': 'eu-de_unittest', 'domain_id': None, 'domain_name': '***MY_BUCKET***'})}))
session = <otcextensions.sdk.imsv2.v2._proxy.Proxy object at 0x7fbdb9895100>
prepend_key = False, base_path = None
    def create(self, session, prepend_key=False, base_path=None):
        # Overriden here to override prepend_key default value
>       return super(Image, self).create(session, prepend_key, base_path
                                         + '/action')
E       TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
/usr/local/lib/python3.9/dist-packages/otcextensions/sdk/imsv2/v2/image.py:91: TypeError
```

--- a/otcextensions/sdk/imsv2/v2/_proxy.py
+++ b/otcextensions/sdk/imsv2/v2/_proxy.py
@@ -22,7 +22,8 @@ class Proxy(proxy.Proxy):
         :param dict attrs: Keyword arguments which will be used to create a
             :class:`~otcextensions.sdk.imsv2.v2.image.Image`
         """
-        return self._create(_image.Image, **attrs)
+        base_path = '/cloudimages/action'
+        return self._create(_image.Image, base_path=base_path, **attrs)
 
     def images(self, **attrs):
         """Retrieve a generator of images

--- a/otcextensions/sdk/imsv2/v2/image.py
+++ b/otcextensions/sdk/imsv2/v2/image.py
@@ -87,6 +87,5 @@ class Image(resource.Resource):
     protected = resource.Body('protected', type=bool)
 
     def create(self, session, prepend_key=False, base_path=None):
-        # Overriden here to override prepend_key default value
-        return super(Image, self).create(session, prepend_key, base_path
-                                         + '/action')
+        # Override here to override prepend_key default value
+        return super(Image, self).create(session, prepend_key, base_path)

--- a/otcextensions/tests/functional/sdk/imsv2/v2/test_image.py
+++ b/otcextensions/tests/functional/sdk/imsv2/v2/test_image.py
@@ -28,7 +28,7 @@ class TestImage(base.BaseFunctionalTest):
             "name": "CentOS-7-x86_64-GenericCloud.qcow2",
             "description": "Create an image from a file in an OBS bucket",
             "image_url":
-                "ims-extensions-test:CentOS-7-x86_64-GenericCloud.qcow2",
+                "ims-test-images:rancheros-openstack.img",
             "os_version": "CentOS 7.0 64bit",
             "min_disk": 40,
             "image_tags": [{"key": "key2", "value": "value2"},

